R/pkg/R/DataFrame.R

```diff
@@ -570,10 +570,17 @@ setMethod("unpersist",
 
 #' Repartition
 #'
-#' Return a new SparkDataFrame that has exactly numPartitions partitions.
+#' There are two different options for repartition
+#' Option 1
+#'   Return a new SparkDataFrame that has exactly numPartitions partitions.
+#' Option 2
+#'   Return a new SparkDataFrame which has as many partitions as the number of unique
```

Comment:
This is not true, the number of partition could still be specified by `numPartitions `, or 200 by default.